src/CMakeLists.txt

```diff
@@ -320,7 +320,7 @@ configure_file(config-keepassx.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/config-keepas
 configure_file(git-info.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/git-info.h)
 
 add_library(autotype STATIC ${autotype_SOURCES})
-target_link_libraries(autotype Qt5::Core Qt5::Widgets)
+target_link_libraries(autotype Qt5::Core Qt5::Network Qt5::Widgets)
```

Comment:
I think this was because `autotype_SOURCES` includes `core/Tools.cpp` (even before my change), which now [`QNetworkCookie`](https://doc.qt.io/qt-5/qnetworkcookie.html), which requires the Qt Network module.

Reply:
Yup this is needed. If I remove it, I get errors like:

```
build] /home/carlo/src/keepassxc/qt-modernize/src/core/Tools.cpp:39:10: fatal error: QNetworkCookie: No such file or directory
[build]    39 | #include <QNetworkCookie>
[build]       |          ^~~~~~~~~~~~~~~~
```